apps/web/src/views/Swap/AkkaSwap/hooks/useApproveCallbackFromAkkaTrade.ts

```diff
@@ -1,10 +1,10 @@
-import { ROUTER_ADDRESS } from 'config/constants/exchange'
+import { ROUTER_ADDRESS_COMMON_AKKA } from 'config/constants/exchange'
 import { ChainId, Currency, CurrencyAmount } from '@pancakeswap/sdk'
 import { useApproveCallback } from 'hooks/useApproveCallback'
 import { useMemo } from 'react'
 
 // wraps useApproveCallback in the context of a swap
 export function useApproveCallbackFromAkkaTrade(inputAmount: CurrencyAmount<Currency>) {
   const amountToApprove = useMemo(() => inputAmount || undefined, [inputAmount])
-  return useApproveCallback(amountToApprove, ROUTER_ADDRESS[ChainId.BITGERT].Akka)
+  return useApproveCallback(amountToApprove, ROUTER_ADDRESS_COMMON_AKKA)
```

Comment:
also here, please stay with ROUTER_ADDRESS and use the current chainId

Reply:
> also here, please stay with ROUTER_ADDRESS and use the current chainId

ok